src/ray/object_manager/object_manager.cc

```diff
@@ -847,7 +845,6 @@ void ObjectManager::RecordMetrics() const {
                                      {{stats::ValueTypeKey, "num_pull_requests"}});
   stats::ObjectManagerStats().Record(profile_events_.size(),
                                      {{stats::ValueTypeKey, "num_profile_events"}});
-  connection_pool_.RecordMetrics();
```

Comment:
replace the old metrics with grpc based implementation, instead of just throwing it away?

Reply:
I'm not sure if it's possible to get the underlying connections w/ grpc though.

Reply:
yes, I will check that and open a dedicated pr for it.

> replace the old metrics with grpc based implementation, instead of just throwing it away?